### PR TITLE
Pashto language localization

### PR DESCRIPTION
**Pull request by isshort**
Added Pashto language localization.

### CODE REVIEW

**Comment by isshort** [comment]
is exist there  any issues?

**Comment by larryaasen** [comment]
I appreciate you adding the Pashto language localization.

In your code, the ps language should appear right after the nl language to keep them alphabetized.

File: CHANGELOG.md
Comment (larryaasen) [inline CHANGELOG.md:20]: What is this change for?

File: CHANGELOG.md
Comment (larryaasen) [inline CHANGELOG.md:93]: What is this change for?

File: CHANGELOG.md
Comment (larryaasen) [inline CHANGELOG.md:103]: What is this change for?

File: CHANGELOG.md
Comment (larryaasen) [inline CHANGELOG.md:127]: What is this change for?

File: CHANGELOG.md
Comment (larryaasen) [inline CHANGELOG.md:135]: What is this change for?

File: README.md
Comment (larryaasen) [inline README.md:523]: What is this change for?

**Comment by larryaasen** [comment]
@isshort If you would like this PR to be merged, you first need to address the comments and the merge conflict.

**Comment by larryaasen** [comment]
@isshort Any update on this?

File: CHANGELOG.md
Comment (isshort) [reply CHANGELOG.md:103]: This was my IDE's automatic changes.

File: CHANGELOG.md
Comment (isshort) [reply CHANGELOG.md:135]: This was my IDE's automatic changes.

File: README.md
Comment (isshort) [reply README.md:523]: This was my IDE's automatic changes.

**Review by mzisshort — APPROVED** [review]

**Comment by larryaasen** [comment]
@isshort I can't merge this PR until you address the comments in this PR.

**Comment by larryaasen** [comment]
@isshort I cannot merge this PR unless the random file changes are removed.

**Comment by isshort** [comment]
> @isshort I cannot merge this PR unless the random file changes are removed.

Okay I'll fix it asap.

**Comment by larryaasen** [comment]
Closed due to inactivity.

**Pull request closed without merging**
